microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -275,3 +282,9 @@ def _process_perf_metric_test_result(self, output: str) -> str:
         if result:
             return result.group(0)
         return ""
+
+    def _save_dmesg_logs(self, log_path: Path) -> None:
+        dmesg_str = self.node.tools[Dmesg].get_output()
```

Comment:
use force_run, otherwise it always uses cache log

Reply:
Done.